<launch>
    <rosparam file="$(find urs100_hardware_interface)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find urs100_hardware_interface)/config/controllers.yaml" command="load"/>
    <rosparam file="$(find urs100_hardware_interface)/config/joint_limits.yaml" command="load"/>

    <node name="urs100_hardware_interface" pkg="urs100_hardware_interface" type="urs100_hardware_interface_node"
          output="screen">
        <remap from="/joint_states" to="/urs100/joint_states"/>
    </node>
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="true" output="screen" ns="/"
          args="
            /urs100/controller/state
            /urs100/controller/position/joint_urs100
        "/>
</launch>
